requirements.txt

```diff
@@ -12,5 +12,6 @@ bert-extractive-summarizer>=0.4.2
 validators~=0.15.0
 nltk>=3.5
 numpy>=1.18.4
+Flask>=1.1.2
 neuralcoref==4.0.0
-spacy==2.1.3
\ No newline at end of file
+spacy==2.1.3
```

Comment:
Flask at the bottom?
What changed in spacy?

Reply:
It's because of merge conflicts that occurred. That's why the sequencing difference. I will add Flask to bottom.